src/main/java/com/flansmod/common/guns/ItemGun.java

```diff
@@ -1077,6 +1084,7 @@ public void onMouseHeld(ItemStack stack, World world, EntityPlayerMP player, boo
         }
     }
 
+    private boolean canClick = true;
```

Comment:
.. this is reversing some of my recent commits I'm pretty sure, please reverse this.